src/VirtualClient/VirtualClient.Actions/StressNg/StressNgExecutor.cs

```diff
@@ -158,10 +181,27 @@ private async Task CaptureMetricsAsync(IProcessProxy process, string commandArgu
 
         private string GetCommandLineArguments()
         {
-            int coreCount = Environment.ProcessorCount;
+            string commandLineArgs = this.CommandLine;
+
+            if (!commandLineArgs.Contains("--cpu"))
```

Comment:
Even in the scenario where we need to use only some % of CPU, is this parameter present in the command line?

Reply:
There the parameter is --cpu-load. --cpu/-c is for cpu count